dask/dask.yaml

```diff
@@ -12,6 +12,7 @@ dataframe:
   parquet:
     metadata-task-size-local: 512  # Number of files per local metadata-processing task
     metadata-task-size-remote: 16  # Number of files per remote metadata-processing task
+  nullable_backend: "pandas"  # Nullable dtype implementation to use
```

Comment:
How do you expect this option (and its default) to interact with the corresponding pandas 2.0 config option? When pandas-2 is released, should the default just correspond to whatever the pandas default is?

For example, it would be nice if we were able to use a test like this for pandas-2:

```python
with pd.option_context("io.nullable_backend", "pyarrow"):
    df = pd.read_parquet("tmp.parquet", engine="pyarrow", use_nullable_dtypes=True)
    ddf = dd.read_parquet("tmp.parquet", engine="pyarrow", use_nullable_dtypes=True)
assert_eq(dd, ddf)
```
Does client vs config options make this a challenge?